lektor_tinymce.py

```diff
@@ -20,8 +21,7 @@
                 txt_elem.classList.add('tinymce-attached');
                 tinymce.init({
                     target: txt_elem,
-                    branding: false,
-                    plugins: 'image link',
+                    {{ tinymce_settings|safe }}
```

Comment:
If you move this to be the last argument, the settings string won't need a comma at the end